tests/ReflectionClassTest.php

```diff
@@ -290,4 +290,37 @@ protected function getGettersToCheck()
 
         return $allNameGetters;
     }
+
+    /**
+     * Tests getName() method
+     *
+     * @dataProvider getClassNamesToCheck
+     *
+     * @param string $className
+     */
+    public function testClassName($className) {
+
+        $this->assertSame(
```

Comment:
Interesting. I the `getName` method result is compared in `testReflectionMethodParity` test, but one of 2 things happens:

1. either `->getName()` result is correct AND `->name` isn't correct
2. or class names list used for testing isn't complete

Likely you should add classes from global namespace to these tests as well.

Reply:
Should we add FQ names for classes and functions as distinct cases in the `@dataProvider`s for `ReflectionClassTest` and `ReflectionFunctionTest`. (We should also similarly add case-insensitivity test cases to the `@dataProvider`s for `ReflectionFunctionTest` and `ReflectionMethodTest` While we're at it.)